main.py

```
@@ -40,27 +49,71 @@ class Movie(BaseModel):
 # --- API Endpoints ---
 
 @app.get("/", summary="Health Check")
 def read_root():
     """Health check endpoint to confirm the API is running."""
-    return {"status": "API is running"}
+    return {"recommendations": recommendations}
+
+# --- Admin Endpoints ---
+
+@app.post("/admin/retrain-models", response_model=Dict[str, Any], tags=["admin"])
+async def retrain_models(current_user: dict = Depends(require_auth)):
```

Ruff (0.13.1)

Unused function argument: `current_user`

(ARG001)